app-text/mandoc/mandoc-1.14.5-r1.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Suite of tools compiling mdoc and man"
+HOMEPAGE="https://mdocml.bsd.lv/"
+SRC_URI="https://mdocml.bsd.lv/snapshots/${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="system-man"
+
+RDEPEND="sys-libs/zlib
+	system-man? ( !sys-apps/man-db )
+"
+DEPEND="${RDEPEND}"
+
+pkg_setup() {
+	use system-man && \
+	[[ "${PORTAGE_COMPRESS}" =~ "gzip"* ]] || \
```

Comment:
`${PORTAGE_COMPRESS}` is a command used for compression, not list of used compression formats or something, so no need for that regular expression. And `p*` would match 0 or more of `p`.

I suggest simply:
`[[ ${PORTAGE_COMPRESS} != gzip ]] && \`

Reply:
done!